infrastructure/code/bundle.go

```diff
@@ -94,6 +95,28 @@ func getIssueLangAndRuleId(issue snyk.Issue) (string, string, bool) {
 	return ruleIdSplit[0], ruleIdSplit[1], true
 }
 
+// isAutofixSupportedForExtension shall call `GetFilters` if necessary and cache the results
+// in the `codeSettings` singleton.
+func (b *Bundle) isAutofixSupportedForExtension(ctx context.Context, file string) bool {
```

Comment:
Non-blocking question: Is bundle really the right place to have this function? Would it maybe be better located as callable function in a different code-related type?

Reply:
Like the autofix-related things gathered into some receiver? That would be good, naturally poses a bit of challenge to me since I need bunch of things like `SnykCodeHttpClient` whose lifecycle I can't precisely reason about, and also access to some of `bundle`s information such as `rootPath` -- and bundle's even less clear regarding their lifecycle...

But I'm more than happy to have a session about the best direction of making further changes, to not mess this part of the codebase.